File: contoh.java/hargabarang.java
import java.util.Scanner;

public class hargabarang {
    public static void main(String[] args) {
        @SuppressWarnings("resource")
        Scanner sc = new Scanner(System.in);
        System.out.println("Masukkan harga barang: ");
        double harga = sc.nextDouble();
        System.out.println("Masukkan jumlah barang: ");
        int jumlah = sc.nextInt();

        double totalHarga = harga * jumlah;

        System.out.println("Total Harga: " + totalHarga);
        if(totalHarga > 100000)
            System.out.println("Status: Diskon 10%");
        else
            System.out.println("Status: Tidak ada diskon");
    }
}

File: contoh.cpp/hargabarang.cpp
#include <iostream>
using namespace std;    
int main() {
    float hargaBarang, diskon, hargaAkhir;
    cout << "Masukkan harga barang: ";
    cin >> hargaBarang;

    if (hargaBarang > 100000) {
        diskon = 0.1 * hargaBarang;
    } else {
        diskon = 0; 
    }

    hargaAkhir = hargaBarang - diskon;

    cout << "Harga Akhir setelah diskon: " << hargaAkhir << endl;

    return 0;
}